crates/plugins/src/manifest.rs

```diff
@@ -1,4 +1,5 @@
 use anyhow::{anyhow, Context, Result};
+use is_terminal::IsTerminal;
```

Comment:
Can we just use `std::io::IsTerminal` now that it's part of the `std`?

Reply:
We could and should.  This is what I get for mashing the "add a use" tooltip.  Thanks!